mcp_servers/mcp_manager.py

```
@@ -235,7 +257,21 @@ async def format_tool_result(result_text: str) -> str:
+            def format_agent_error(error):
+                try:
+                    error_str = str(error)
+                    if any(ord(char) > 127 for char in error_str):
+                        error_str = error_str.encode(
+                            "utf-8",
+                            errors="replace",
+                        ).decode("utf-8")
+                except UnicodeError:
+                    error_str = repr(error)
+                return (
+                    "学术智能体（Academic Agents）调用出错: "
```

Ruff (0.16.0)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)